### DISCUSSION TITLE
How to check status SSH port?

### DISCUSSION THREAD

**Discussion by quocthinhvo** [upptime/upptime#482] [Q&A]
I use tcp-ping, but this always show UP, but this address not exists.
![image](https://user-images.githubusercontent.com/46624702/144791955-50e2b1b4-44f0-4b32-8596-c3e251aef36e.png)

**Comment by thomasmerz** [comment]
See my **discussion** https://github.com/upptime/upptime/discussions/276 from february 2021.
I opened an **issue** #493 now for this.
